api/domain/customersDomain.js

```diff
@@ -44,23 +33,20 @@ exports.delete = async (customerId) => {
 exports.getAll = async () => {
     let fetchedRows = await customerRepo.getAll();
 
-    if (fetchedRows === 0) {
-        throw new Error('No customers found.');
+    //IF TABLE IS EMPTY STILL RETURNS INSTEAD OF THROWING AN ERROR
```

Comment:
remove comment

Reply:
Comments in my code don't exist ;))))))))))))))))))))))